bitcoin/CHANGELOG.md

```diff
@@ -1,7 +1,102 @@
+- Add `Weight::from_vb_weight function` function (added in [#2206](https://github.com/rust-bitcoin/rust-bitcoin/pull/2206),
+  renamed in [#2228](https://github.com/rust-bitcoin/rust-bitcoin/pull/2228)
+- Implement `LowerHex` and `UpperHex` for `CompactTarget` [#2221](https://github.com/rust-bitcoin/rust-bitcoin/pull/2221)
+- Add conversions from TXIDs to merkle nodes [#2227](https://github.com/rust-bitcoin/rust-bitcoin/pull/2227)
+- Add `transaction::effective_value()` function [#2230](https://github.com/rust-bitcoin/rust-bitcoin/pull/2230)
+- Improve `LockTime`s and related types:
+   - Improve relative locktime API [#2549](https://github.com/rust-bitcoin/rust-bitcoin/pull/2549)
+   - Implement `ordered::ArbitraryOrd` for absolute and relative locktimes, introducing the "ordered" feature:
+     - [#2581](https://github.com/rust-bitcoin/rust-bitcoin/pull/2581)
+     - [#2248](https://github.com/rust-bitcoin/rust-bitcoin/pull/2248)
+- Add and use custom `ArrayVec` 
+   - [#2287](https://github.com/rust-bitcoin/rust-bitcoin/pull/2287)
+   - [#2286](https://github.com/rust-bitcoin/rust-bitcoin/pull/2286)
+   - [#2310](https://github.com/rust-bitcoin/rust-bitcoin/pull/2310)
+- Add the implementation of `Display` for `transaction::Version` [#2309](https://github.com/rust-bitcoin/rust-bitcoin/pull/2309)
+- Add functionality to serialize signatures to a writer [#2392](https://github.com/rust-bitcoin/rust-bitcoin/pull/2392)
+- Add unchecked integer operations to `Amount` and `SignedAmount` (add, sum etc.) [#2436](https://github.com/rust-bitcoin/rust-bitcoin/pull/2436)
+- Add methods `p2wpkh_script_code` to `Script` and `ScriptBuf` [#2445](https://github.com/rust-bitcoin/rust-bitcoin/pull/2445)
+- Improve API for signing Taproot inputs in PSBTs [#2458](https://github.com/rust-bitcoin/rust-bitcoin/pull/2458)
+- Add `TapNodeHash` getter method on `TapTree` and `NodeInfo` [#2467](https://github.com/rust-bitcoin/rust-bitcoin/pull/2467)
+- Add hex parsing to pow types [#2514](https://github.com/rust-bitcoin/rust-bitcoin/pull/2514)
+- Implement `From<core::convert::Infallible>` for errors [#2516](https://github.com/rust-bitcoin/rust-bitcoin/pull/2516)
+- Add ServiceFlags::P2P_V2 [#2555](https://github.com/rust-bitcoin/rust-bitcoin/pull/2555)
+
+## Other API breaking changes
+
+- Require `io::BufRead` instead of `io::Read` in consensus decode trait [#2240](https://github.com/rust-bitcoin/rust-bitcoin/pull/2240)
+- Make `Payload` and `AddressEncoding` private [#1979](https://github.com/rust-bitcoin/rust-bitcoin/pull/1979)
+- Remove `bech32` from the public API [#2381](https://github.com/rust-bitcoin/rust-bitcoin/pull/2381)
+- Make from_hex inherent for byte-like types [#2491](https://github.com/rust-bitcoin/rust-bitcoin/pull/2491)
+- Remove the FromHexStr trait [#2492](https://github.com/rust-bitcoin/rust-bitcoin/pull/2492)
+
+## Error handling
+
+- Split `Prevouts` errors out into specific error types 2042
```

Comment:
nit: Are you also going to link these? The standalone numbers look strange, we can also remove them completely.

Reply:
Drats, I missed these - this is what the looked like before I hit them with my emacs macro to fill in the links. This was an oversight, thanks for catching it.